e2e-tests/cypress/support/locators/mentors_page/mentors_page.locate.js

```diff
@@ -39,17 +43,59 @@ class mentorsLocatorManager {
   };
 
   //Search section
-  validateKeywordInput = () => {
+  getMentorsSearchInput = () => {
+    return cy
```

Comment:
💅  This is kind of preference for me, but to keep one line assertion would be way better and reduce the amount of lines in the test file, which makes easier to main the code. 

```suggestion
    return cy..get(KEYWORD_INPUT).shouldBeVisible();
```

For you It is just a tip, you don't need to take care of that as part of this PR, but would be nice to consider that in next ones. 

In case does not fit in 1 one line then you should break, otherwise it is easier to ready and follow the tests with single line. (As you did on the lines below)

Reply:
Sorry, I am not sure I understand you correctly. You mean formatting issue?

Reply:
Yes, in some places you are breaking in 3 lines and in others you are not. My suggestion is no not break unless it does not fit into the recommended by the VS Code editor (or the editor you are using)

Reply:
Yeah, everything I do is selecting all content of the file, right-click and Format document in VS Code. I'll take care of it more attentively how it works in the next PRs